test-projects/access-control/index.js

```diff
@@ -144,7 +158,19 @@ listAccessVariations.forEach(createListWithDeclarativeAccess);
 module.exports = {
   keystone,
   apps: [
-    new GraphQLApp(),
-    new AdminUIApp({ name: projectName, adminPath: '/admin', authStrategy }),
+    new SchemaRouterApp({
+      routerFn: () => 'testing',
+      apps: {
+        seeding: createGraphQLApp('seeding'),
+        testing: createGraphQLApp('testing'),
```

Comment:
```suggestion
        seeding: new GraphQLApp({ schemaName: 'seeding' }),
        testing: new GraphQLApp({ schemaName: 'testing' }),
```
No need for the extra function definition here.